### DISCUSSION TITLE
I've been waiting for something like this

### DISCUSSION THREAD

**Discussion by stevenstallone** [morning4coffe-dev/project-sbs#114] [General]
I've been wanting a program like this for a while, there are two android apps that are comparable, one is "Tilla" and the other is Kapersky's app which is surprisingly hard to find unless you know the name "SubsCrab". The problem is that they don't have a windows version. What I like about both is that you can have logo icons that can help you easily identify the subscription. Another thing I like about SubsCrab is that it can display your subscriptions as a calendar which can help you visualize the time period for a subscription. Maybe these are things that could be added in the future.

What astonishes me is that Microsoft doesn't connect the calendar at the bottom right of the taskbar to the outlook calendar or something, it's simply used to tell the date only, when it could be used as a quick access panel you can pull up to not only to check the date but also check something marked in your outlook calendar or potentially see a subscription like in this app.

One thing I am curious about, is that this app is almost indistinguishable from a native Microsoft app like Microsoft "To Do" and you have to login to your Microsoft account in order for your subscriptions to sync across devices. I was wondering exactly how that works since this is not a Microsoft app, also I was wondering if it would be possible to implement a custom import export file or pull from a file stored in your own cloud without having to login. It does say when you are not logged in that the data is stored locally, is that accessible and able to be backed up at the moment?

One thing that just occurred to me is that since this is such a heavily Microsoft orientated app, it would be a cool idea to have a companion Game Bar Widget where you could easily see your subscriptions by opening the Game Bar overlay. I'm a big fan of Game Bar since I realized how easy it is to control volume from it.

It would also be nice if you could customize your own categories or hide the each of the existing ones as it's a bit cluttered.

Anyway, thanks for making this, hopefully it blows up and becomes a program everyone uses.

**Comment by morning4coffe-dev** [comment]
Hey @stevenstallone,

Thank you so much for your wonderful comment, it made my day. 😊 I'm thrilled to hear that you've been enjoying the app and that it's filling a gap that you've been looking to fill.

I am always looking for ways to improve, so I’ll definitely add your ideas to my backlog for future updates. My current priority is bringing this app to other platforms and stores as well to attract as many users as possible.

The reason why it might seem similar to apps coming directly from Microsoft is that it uses Fluent design, and it was actually created as my final project in a student program supported by Microsoft. I’m glad to hear that the design resonates with you!

As for your request regarding sign-in options, you're not alone—many users have asked for this, and it's definitely on my roadmap with the multiplatform support. I’m planning to implement more sign-in options soon to give users more flexibility.

Importing/exporting is a great suggestion. Currently, when you're signed in, your data is automatically saved in a dedicated folder within your OneDrive account. The app doesn’t have access to any other files in your OneDrive, just the folder it creates for storing your subscription data. When data is stored locally without logging in, it’s not directly accessible to users at the moment. However, when the the import/export feature is implemented, it will allow users to manually back up and manage their data more easily without needing to dive into the app's backend.

If you’d like to help out, definitely be sure to star the GitHub page, leave a review on the Microsoft Store, and spread the word to your friends. Every bit of support helps!

Thanks again for your thoughtful feedback and support! Your suggestions are invaluable, and I’m excited to keep improving the app. If you have any more ideas or questions, feel free to reach out anytime.

- Dominik
